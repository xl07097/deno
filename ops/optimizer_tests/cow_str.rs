fn op_cow_str(c: Cow<'_, str>) {
  // ...
}
