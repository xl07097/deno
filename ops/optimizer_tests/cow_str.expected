=== Optimizer Dump ===
returns_result: false
has_ref_opstate: false
has_rc_opstate: false
has_fast_callback_option: false
needs_fast_callback_option: true
fast_result: Some(Void)
fast_parameters: [V8Value, SeqOneByteString]
transforms: {0: Transform { kind: SeqOneByteString(Cow), index: 0 }}
is_async: false
fast_compatible: true
